kustomize/overlays/sandbox/propeller/config.yaml

```diff
@@ -64,3 +64,4 @@ tasks:
       - container
       - sidecar
       - spark
+      - k8s-array
```

Comment:
I think we should also add it to the end2end tests.

And while we are at it, maybe also update all versions of components?